docs/source/transcoding.md

```diff
@@ -84,7 +86,19 @@ you. The important thing is that you keep the node running.
 
 **I get an error that looks something like "failed to estimate gas needed: gas required exceeds allowance or always failing transaction".**
 
-- This is because the gas estimator is giving incorrect estimates.  To fix it, you can manually pass in a gas limit using `-gasLimit`.  For example, `$ livepeer -transcoder -publicIP x.x.x.x -gasLimit 400000`.
+- This is because the gas estimator is giving incorrect estimates.  To fix it, you can manually pass in a gas limit using `-gasLimit`.  For example, `$ livepeer -transcoder -gasLimit 400000`.
+
+**What does being 'publicly accessible' mean? Can I run a transcoder from home?**
+
+- The transcoder should be reachable by broadcasters via the public IP and port that is set during transcoder configuration. Transcoders will not be able to serve the Livepeer network if they are behind a NAT (eg, a home router). If this is the case, special accommodations must be made for the transcoder, such as port forwarding or putting the the transcoder in the DMZ. The only port that is required to be public is the one that was set during the transcoder registration step (default 8935). Be aware that there are many risks to running a public server. Only set up a transcoder if you are comfortable with managing these risks.
+
+**What is the Service URI? Does this need to be an IP?**
+
+The Service Registry acts as a discovery mechanism to allow broadcasters to look up the addresses of transcoders on the network. Transcoders register their Service URI at configuration time; this is submitted to the blockchain as a standalone transaction. While the configuration tool only asks for your IP:port, the URI stored on the blockchain in the form of `https://IP:port`. Transcoders are expected to provide a consistent and reliable service, so IPs here *should* remain static. However, a host (DNS) name is also allowed for the service URI to give transcoders some flexibility.
```

Comment:
This paragraph makes it sounds like IP is encouraged over hostname. Is that really the case? 

I suspected that by setting a hostname it would make it easier to configure things like DDoS protection using Cloudflare (or similar), or even to loadbalance to various backend orchestrators by region via smart dns? We don't have to point these out here, but are there downsides to using a hostname?

Reply:
Yeah, actually I had a good discussion with @darkdarkdragon about load balancing in particular yesterday. Load balancing is actually one of the intended uses of an hostname, however an orchestrator would have to be extremely careful with that setup to avoid doing double work and making double claims. So it'd definitely be an advanced feature.

Probably depends on who the intended audience is here. Seasoned devops folks would be able to understand the implications of a transcoder URI, so I wanted to err on the side of being explicit about the expected behavior of an IP for folks that don't have as much experience. Is there some other phrasing that might sound better?